spec/jobs/course/material/download_job_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Course::Material::ZipDownloadJob do
+  let(:instance) { create(:instance) }
+  with_tenant(:instance) do
+    let(:folder) { create(:folder) }
+    subject { Course::Material::ZipDownloadJob }
+
+    it 'can be queued' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.